examples/control-elements/media-chrome-menu-button.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta name="viewport" content="width=device-width">
+  <title>Media Chrome Captions Usage Examples</title>
+  <script type="module" src="../../dist/index.js"></script>
+  <script type="module" src="../../../dist/media-chrome-listbox.js"></script>
+  <script type="module" src="../../../dist/media-chrome-listitem.js"></script>
+  <script type="module" src="../../../dist/media-captions-listbox.js"></script>
+  <script type="module" src="../../../dist/media-chrome-menu-button.js"></script>
+  <style>
+    media-captions-listbox {
+      display: block;
+      height: fit-content;
+    }
+
+    media-chrome-listitem {
+      position: relative;
+      line-height: 1.5em;
+    }
+    media-chrome-listitem[aria-selected="true"]::before {
+      content: '✅';
+      position: absolute;
+      margin-left: -1.5em;
+    }
+
+    .examples {
+      margin-top: 20px;
+    }
+  </style>
+</head>
+<body>
+  <main>
+    <h1>Media Chrome Standard Video Usage Example</h1>
+    <media-chrome-menu-button>
+      <media-chrome-listitem>lorem</media-chrome-listitem>
```

Comment:
You can either do list items in the default slot or swap the listbox? That's pretty cool.

Reply:
Yup, inspired by selectmenu where you can either use the component directly or slot your own if you wanted/needed more functionality.